packages/app/package.json

```diff
@@ -33,7 +34,7 @@
     "@vercel/webpack-asset-relocator-loader": "1.7.0",
     "css-loader": "^6.0.0",
     "electron": "16.0.6",
-    "eslint": "^8.0.1",
+    "eslint": "^7.32.0",
```

Comment:
We have master eslint in the root - this causes problems because eslint 7 and 8 are incompatible ... let's stick with 7 until we are able to update ALL packages to version 8